tasks/skylab/StarAlign.wdl

```diff
@@ -512,7 +512,7 @@ task MergeStarOutput {
 
   runtime {
     docker: docker
-    memory: "${machine_mem_mb} MiB"
+    memory: "${machine_mem_mb} GiB"
```

Comment:
If we change to GiB, it might make sense to change the varialble name machine_mem_gb so as not to confuse users. Brian has been confused about this on previous runs.

Reply:
That makes sense. Thanks for pointing that out!